app.py

```diff
@@ -16,60 +15,61 @@
 
 tags = {
     "Project": "veda",
-    "Owner": config.owner,
+    "Owner": app_settings.owner,
     "Client": "nasa-impact",
-    "Stack": config.stage,
+    "Stack": app_settings.stage,
     "GitCommit": git_sha,
     "GitTag": git_tag,
 }
 
 app = cdk.App()
-stack = AuthStack(app, f"veda-auth-stack-{config.stage}")
-
-# Create a data managers group in user pool if data managers role is provided
-if data_managers_role_arn := config.data_managers_role_arn:
```

Comment:
We need a strategy to support assuming a pre-defined data managers role somehow to ensure that we will still have assumed role access to DAAC buckets, etc.